src/main/java/com/minecolonies/coremod/proxy/ClientProxy.java

```diff
@@ -104,6 +104,7 @@ public void openBuildToolWindow(@Nullable final BlockPos pos)
     @Override
     public void registerRenderer()
     {
+        // ModelLoader.setCustomModelResourceLocation todo from preinit
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)